src/components/Auth/Auth.tsx

```diff
@@ -47,8 +71,8 @@ export function RequireAuth() {
 }
 
 export function RequireNoAuth() {
-  let auth = useAuth();
-  let location = useLocation();
+  const auth = useAuth();
+  const location = useLocation();
 
   if (auth.user) {
     return <Navigate to="/" state={{ from: location }} replace />;
```

Comment:
Now `/login` is the only route that doesn't require authentication. If in the future we add other pages that don't require auth, then this would cause unwanted redirects for logged in users. 

Not an issue for now, but I wonder if it would be better to move this redirect to the Login component, since it is specific to it?